examples/managed_instance_group/variables.tf

```diff
@@ -38,30 +35,37 @@ variable "mig_instance_count" {
 
 variable "image" {
   description = "The Docker image to deploy to GCE instances"
+  default     = "gcr.io/google-samples/hello-app:1.0"
 }
 
 variable "image_port" {
   description = "The port the image exposes for HTTP requests"
```

Comment:
Types should be specified throughout `variables.tf`

Reply:
@Jberlinsky 
```
$ grep type  examples/*/variables.tf | wc -l
5
$ grep variable examples/*/variables.tf | wc -l
26
```

Reply:
@kopachevsky @Jberlinsky 
[Fixed](https://github.com/terraform-google-modules/terraform-google-container-vm/compare/21506b0e8d88a8bee72d02dc63094b8276ccd288..b6d0f962d7a2074d4392d31a3368b8c6808df77b)